pygromos/data/simulation_parameters_templates/vac_emin.imd

```diff
@@ -24,7 +24,7 @@ FORCE
 # 	 BONDS 	 ANGLES 	 IMPROPER 	 DIHEDRAL 	 ELECTROSTATIC 	 VDW
  	 1 	 1 	 1 	 1 	 1 	 1 	 
 # 	 NEGR 	 NRE
- 	 1 	 71 	 
+ 	 1 	 1 	 
```

Comment:
same here :)

Reply:
i did not intentionally change this - can you just keep the original version?

Reply:
jap I can do that :)